cal_pomodoro_stats_name_hangout_emily_cal.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
not blocking but we should move to checking in paired .py scripts rather than the notebooks... I'll take an AI to set this up